modules/nux/site.js

```diff
@@ -52,13 +52,12 @@ var display_final_nux_step = function(res) {
                 [{'name': 'hm_ajax_hook', 'value': 'ajax_imap_accept_special_folders'},
                 {'name': 'imap_server_id', value: res.nux_server_id},
                 {'name': 'imap_service_name', value: res.nux_service_name}],
-                function(res) {
-                    window.location.href = "?page=servers";
+                function() {
+                    Hm_Utils.redirect();
```

Comment:
Don't we want to keep the redirect to servers page? Otherwise, it simply reloads the nux page which doesn't make sense - user should be able to see the added server on servers page - I think that's the original idea. Do I miss something?